libraries/standard/http/utest/http_utest.c

```diff
@@ -249,3 +334,260 @@ void test_Http_InitializeRequestHeaders_insufficient_memory()
 }
 
 /* ========================================================================== */
+
+void test_Http_AddRangeHeader_Invalid_Params( void )
```

Comment:
nitpick: I think the convention in AFR is to have the description of the test in lowercase, but it's really not a big deal for me: `_invalid_params`

Reply:
Don't think that AFR has a consistent casing pattern. This file is using camel casing for the description
https://github.com/aws/amazon-freertos/blob/master/libraries/abstractions/pkcs11/utest/iot_pkcs11_mbedtls_utest.c#L1848